scripts/workers/code_upload_worker_utils/make_submission.sh

```diff
@@ -1,6 +1,6 @@
 #!/bin/sh
-submission_url="$EVALAI_API_SERVER/api/jobs/challenge/$CHALLENGE_PK/challenge_phase/$PHASE_PK/submission/$SUBMISSION_PK"
-submission_curl_base_request="curl --location --request PATCH '$submission_url' --header 'Authorization: Bearer $AUTH_TOKEN'"
+submission_url="$EVALAI_API_SERVER/api/jobs/challenge/$CHALLENGE_PK/update_submission/"
+submission_curl_base_request="curl --location --request PATCH '$submission_url' --header 'Authorization: Bearer $AUTH_TOKEN' -F 'submission=$SUBMISSION_PK'"
```

Comment:
@savish28 shouldn't this `-F 'submission=$SUBMISSION_PK'` be part of request body?  Have you tested by making a API call locally?

Reply:
@Ram81 It is part of the request body itself, the F stands for Form Data. Yes, I have tested these changes locally and the file reflected on the submission model. Verified from django admin. We are eventually using reques.data and it contains submission and the submission_input_file